### ISSUE TITLE
feature request: Sort by amount of premades:

### ISSUE THREAD

**Issue by madtoad14** [bugdone/headshotbox#114]
Awesome tool first of all.
It'd be cool to see your stats when you solo queue compared to how you do with x amount of premades. 
Also atm the 'round type' filter option, doesn't update the far left table with win % etc.

**Comment by clugg** [comment]
I don't believe demo parsing offers the ability to determine who you were lobbied with this, but there's also the possibility that it detects the amount of people on your team that are currently in your friends list for a similar effect.

**Comment by madtoad14** [comment]
Alright, i thought it could just work like the feature where you can filter games by specific teammates, i have very little knowledge of how this works tho, so shame if it's not possible

**Comment by bugdone** [comment]
Filtering by the amount of friends was already requested in #80.

The round win % is not computed because I was too lazy to do it. Will do.

About filtering CT/T side: you can already filter by side using the Rounds filter. There's also Charts tab -> Round Win %.
